backend/backend/handlers/workflows/executeWorkflow.py

```diff
@@ -35,12 +34,27 @@ def get_pipelines(databaseId, pipelineId):
     )
     return response['Items']
 
-def launchWorkflow(bucketName, key, workflow_arn, asset_id, workflow_id, database_id):
+def launchWorkflow(asset, bucketName, key, workflow_arn, workflow_id, database_id):
     print("Launching workflow with arn: ", workflow_arn)
+    asset_id = asset['assetId']
+    metadata = get_metadata(database_id, asset_id)[0]
+    print(metadata)
```

Comment:
Is this for debugging or real logs? if for real logs, let's use descriptive log messages.